### PR TITLE
Update the documentation reference

### PR DESCRIPTION
**Pull request by emmanuel-ferdman**
### Summary
PR #248 moved the location of `drift-detect.md`. This PR adjusts source to point to: https://lyft.github.io/cartography/usage/drift-detect.html

### Checklist

Provide proof that this works (this makes reviews move faster). Please perform one or more of the following:
- [ ] Update/add unit or integration tests.
- [ ] Include a screenshot showing what the graph looked like before and after your changes.
- [ ] Include console log trace showing what happened before and after your changes.

If you are changing a node or relationship:
- [ ] Update the [schema](https://github.com/lyft/cartography/tree/master/docs/root/modules) and [readme](https://github.com/lyft/cartography/blob/master/docs/schema/README.md).

If you are implementing a new intel module:
- [ ] Use the NodeSchema [data model](https://lyft.github.io/cartography/dev/writing-intel-modules.html#defining-a-node).

### CODE REVIEW

File: cartography/driftdetect/cli.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def _build_parser(self):
                 'graph database and reports the deviations.'
             ),
             epilog='For more documentation please visit: '
-                   'https://github.com/lyft/cartography/blob/master/docs/drift-detect.md',
+                   'https://github.com/lyft/cartography/blob/master/docs/usage/drift-detect.md',
```
Comment (chandanchowdhury) [inline cartography/driftdetect/cli.py:33]: Should point to `https://lyft.github.io/cartography/usage/drift-detect.html`

**Review by chandanchowdhury — CHANGES_REQUESTED** [review]

**Comment by chandanchowdhury** [comment]
Hey @emmanuel-ferdman good catch, thank you!

@achantavy Should these links point to Docs site or MD files in Github?

https://github.com/lyft/cartography/blob/master/docs/root/usage/drift-detect.md
vs
https://lyft.github.io/cartography/usage/drift-detect.html

**Comment by emmanuel-ferdman** [comment]
@chandanchowdhury @achantavy updated the PR. Thanks for the review

File: cartography/driftdetect/cli.py [hunk 2]
```diff
@@ -30,7 +30,7 @@ def _build_parser(self):
                 'graph database and reports the deviations.'
             ),
             epilog='For more documentation please visit: '
-                   'https://github.com/lyft/cartography/blob/master/docs/drift-detect.md',
+                   'https://lyft.github.io/cartography/usage/drift-detect.html',
```
Comment (chandanchowdhury) [inline cartography/driftdetect/cli.py:33]: Apologies for the previous change request, I should have checked with Alex first!

Please point to 
https://github.com/lyft/cartography/blob/master/docs/root/usage/drift-detect.md

**Review by chandanchowdhury — DISMISSED** [review]

File: cartography/driftdetect/cli.py (hunk 2 above)
Comment (achantavy) [reply cartography/driftdetect/cli.py:33]: It's good right now

**Review by achantavy — APPROVED** [review]
Thanks for the fix

**Comment by achantavy** [comment]
There's something weird going on with the CI, I'll ask for help

**Pull request merged**